CONTRIBUTING.md

```diff
@@ -0,0 +1,188 @@
+# ✨ Contributing to this project
```

Comment:
I would name this "Contributing Guide" instead

Reply:
please also run prettier on the entire file

Reply:
Done